model/Streams.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../database/db-connection');
+
+class Streams {
+    constructor(tablename) {
+        this.tablename = tablename;
+    }
+
+    getAll(callback) {
+
+        db.query(`SELECT * FROM ${this.tablename}`, (error, streams) => {
+            if (error) {
+              console.log('Error - could not get redeemables ', error)
+              return callback(error);
+            }
+            return callback(null, streams);
+          });       
+    }
+}
+
+module.exports = new Streams('Streams');
```

Comment:
This class and Redeemables seem like they have the same functionality. Consider

```suggestion
class TableGetter {
/* * * * */
const Streams = new TableGetter('Streams');
const Redeemables = new TableGetter('Redeemables');
module.exports = { Streams, Redeemables };
```

Reply:
Yes sir .. will consider on a backend refactor